src/vm/fieldmarshaler.cpp

```diff
@@ -1229,6 +1242,26 @@ BOOL IsStructMarshalable(TypeHandle th)
 }
 
 
+//=======================================================================
+// This function returns TRUE if the field passed in is a fixed buffer.
+// In all other cases it will return FALSE. 
+//=======================================================================
+BOOL IsFixedBuffer(mdFieldDef field, IMDInternalImport* pInternalImport)
+{
+    CONTRACTL
+    {
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_ANY;
+        PRECONDITION(pInternalImport != NULL);
+    }
+    CONTRACTL_END;
+
+    HRESULT hRESULT = pInternalImport->GetCustomAttributeByName(field, g_FixedBufferAttribute, NULL, NULL);
```

Comment:
Please change the name to `hr` or `hResult`.

Reply:
I would prefer hr. since most of our code use hr. only 1 instance use hResult.